scripts/azcli-integration/main.tf

```diff
@@ -1,5 +1,10 @@
+terraform {
```

Comment:
does it work with terraform 0.12? :) plenty of customers still use two-release old version.